Add PDF download functionality and watermark to agreement printout

SudiptaPaul-31 opened the pull request:
resolves #74 

- Add a watermark of custos logo to agreement pdf when print/download option is triggered

faytey reviewed (changes requested):
- [x] Awesome work, kindly add a screenshot or short video of the implemented output

Goodness5 reviewed (approved):
please add a screenshot of your implementations so i can merge

SudiptaPaul-31 commented:
@Goodness5 Not able to add wallet on custos that's why not able to send the ss I texted you on tg as shown on the image. This issue was from the day 1 thats why not able to share screenshot(s) synced the code still the problem persists

![image](https://github.com/user-attachments/assets/4acfb29a-078d-43ed-a5a2-60ad61e603b6)

Goodness5 reviewed (changes requested):
I'm unable to click on the print agreement buttons in the deployment preview it's unresponsive on every page i.e the agreement cards and the agreement slug page, Also add a screenshot of your changes please

SudiptaPaul-31 committed:
Updated watermark

SudiptaPaul-31 committed:
Updated fix

SudiptaPaul-31 committed:
updated watermark

SudiptaPaul-31 commented:
Updated Changes:

- Added an option to download or print pdf
- Added watermark on the pdf 

## Screenshot(s)

![image](https://github.com/user-attachments/assets/22e91f04-6140-4bef-9099-65e35f898af6)

![image](https://github.com/user-attachments/assets/44fb5b55-3ced-430d-80d4-971647f68aa7)
